### PR TITLE
[compositor] Implement privateGetSetupActions method call

### PR DESCRIPTION
**Pull request by spiiroin**

### CODE REVIEW

**Review by markolemmetty — APPROVED** [review]

File: src/compositor/lipstickcompositor.cpp [hunk 1]
```diff
@@ -71,7 +71,10 @@ LipstickCompositor::LipstickCompositor()
     , m_mceNameOwner(new QMceNameOwner(this))
     , m_sessionActivationTries(0)
 {
-    setColor(Qt::black);
+    QGuiApplication::platformNativeInterface()->nativeResourceForIntegration("DisplayOn");
+    setColor(Qt::magenta);
```
Comment (rainemak) [inline src/compositor/lipstickcompositor.cpp:75]: This could be behind an environment variable so that it's easier to use for testing purposes.

Comment (spiiroin) [reply src/compositor/lipstickcompositor.cpp]: Done like that now

**Comment by spiiroin** [comment]
Rebased, updated Jolla (c) spans, and modified to use opt-in via environment for controlling the magenta debug display.

Draft status removed.

**Review by rainemak — APPROVED** [review]

**Comment by spiiroin** [comment]
Rebased

**Pull request merged**